cmd/grapi-gen-type/di/wire_gen.go

```diff
@@ -0,0 +1,36 @@
+// Code generated by Wire. DO NOT EDIT.
+
+//go:generate wire
+//+build !wireinject
+
+package di
+
+import (
+	"github.com/izumin5210/grapi/pkg/cli"
+	"github.com/izumin5210/grapi/pkg/gencmd"
+	"github.com/izumin5210/grapi/pkg/grapicmd"
+	"github.com/izumin5210/grapi/pkg/protoc"
+)
+
+// Injectors from wire.go:
+
+func NewApp(command *gencmd.Command) (*App, error) {
+	ctx := gencmd.ProvideCtx(command)
```

Comment:
**[unparam]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
ProvideCtx not declared by package gencmd